gtsam/hybrid/tests/testHybridBayesTree.cpp

```diff
@@ -32,6 +32,25 @@ using noiseModel::Isotropic;
 using symbol_shorthand::M;
 using symbol_shorthand::X;
 
+/* ****************************************************************************/
+// Test multifrontal optimize
+TEST(HybridBayesTree, OptimizeMultifrontal) {
+  Switching s(4);
+
+  Ordering hybridOrdering = s.linearizedFactorGraph.getHybridOrdering();
+  HybridBayesTree::shared_ptr hybridBayesTree =
+      s.linearizedFactorGraph.eliminateMultifrontal(hybridOrdering);
+  HybridValues delta = hybridBayesTree->optimize();
```

Comment:
Why would I only get continuous values here?

Reply:
You should be getting both. Probably should test for `delta.discrete()` as well.